cameraview/src/main/java/com/otaliastudios/cameraview/engine/CameraBaseEngine.java

```diff
@@ -70,6 +70,7 @@
     @SuppressWarnings("WeakerAccess") protected boolean mPictureMetering;
     @SuppressWarnings("WeakerAccess") protected boolean mPictureSnapshotMetering;
     @SuppressWarnings("WeakerAccess") protected float mPreviewFrameRate;
+    @SuppressWarnings("WeakerAccess") protected boolean mPreviewFrameRateExact;
```

Comment:
Since you have another change to do, I'd also ask you to make this private. Engines can use the getter now.